test/face/wind/weap/kind/test_catalyst.py

```diff
@@ -17,6 +17,7 @@
         pytest.param("Sacrificial Jade", 4, "Level 80/90 (Rank 6)", 427, WeaponStatType.critical_rate_perc, 33.5, 5, id="face.wind.rule: Configuring weapon - Catalyst - Sacrificial Jade"),
         pytest.param("Waveriding Whirl", 4, "Level 80/90 (Rank 6)", 427, WeaponStatType.energy_recharge_perc, 55.9, 5, id="face.wind.rule: Configuring weapon - Catalyst - Waveriding Whirl"),
         pytest.param("A Thousand Floating Dreams", 5, "Level 80/90 (Rank 6)", 506, WeaponStatType.elemental_mastery, 241, 5, id="face.wind.rule: Configuring weapon - Catalyst - A Thousand Floating Dreams"),
+        pytest.param("Starcaller's Watch", 5, "Level 80/90 (Rank 6)", 506, WeaponStatType.elemental_mastery, 241, 5, id="face.wind.rule: Configuring weapon - Catalyst - Starcaller's Watch"),
```

Comment:
This is an unrelated change.

I will make an exception this time, but please make a separate pull request for unrelated changes while ideally pointing towards the issue ticket where it was reported.